src/app/ReviewService.Abstractions/IReviewService.Extensions.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using ReviewService.Abstractions;
+
+namespace ReviewService;
+
+/// <summary>
+/// Extensions of <see cref="IReviewService{TReviewSettings}"/>.
+/// </summary>
+public static class IReviewServiceExtensions
```

Comment:
Remove the `I` prefix.
https://stackoverflow.com/a/2700711

Reply:
Wasn't sure and based my decision on 
![image](https://github.com/nventive/UnoApplicationTemplate/assets/57812285/b6c7b8ae-c5d2-485b-9760-70b4716cf15d)
😭

Reply:
Should I move it into the other project? Doesn't really fit abstractions anymore.

Reply:
Keep it in Abstractions so that any implementation benefits from the extensions.
I suppose those would be in a .Extensions package, but that's not what we've done with our other recipes until now and that's probably not worth the overhead.